src/conversions/num_bigint.rs

```diff
@@ -93,6 +113,23 @@ macro_rules! bigint_conversion {
                     PyObject::from_owned_ptr(py, obj)
                 }
             }
+
+            #[cfg(Py_LIMITED_API)]
+            fn to_object(&self, py: Python<'_>) -> PyObject {
+                let bytes = $to_bytes(self);
+                let bytes_obj = PyBytes::new(py, &bytes);
+                let kwargs = if $is_signed > 0 {
+                    let kwargs = PyDict::new(py);
+                    kwargs.set_item(intern!(py, "signed"), true).unwrap();
+                    Some(kwargs)
+                } else {
+                    None
+                };
+                py.get_type::<PyLong>()
+                    .call_method("from_bytes", (bytes_obj, "little"), kwargs)
+                    .expect("TODO")
```

Comment:
How should I handle errors?

Reply:
For now `.expect()` is ok , we will eventually improve this in #1813, it's helpful if you could add a `// FIXME #1813 or similar` here.